app/grants/views.py

```diff
@@ -1840,6 +1841,9 @@ def bulk_fund(request):
             'text': _('Funding for this grant was successfully processed and saved.'),
             'success': True
         })
+        batch_grants_mail.append(grant_id)
+
+    thank_you_for_supporting(batch_grants_mail, profile)
```

Comment:
this won't work @corydickson as subscriptions are created by a celery task
`process_grant_contribution.delay(grant_id, grant.slug, profile.pk, payload)`

This means when you try to contribute to 2-3 grants
It causes the email to be sent before the subscription objects have been created!

Hence we don't have the subscription object while we send the mail causing it to error out

<img width="1238" alt="Screenshot 2020-11-30 at 12 56 34 PM" src="https://user-images.githubusercontent.com/5358146/100580955-e0425600-330c-11eb-9dcf-5304bd921c2b.png">
<img width="1041" alt="Screenshot 2020-11-30 at 12 56 51 PM" src="https://user-images.githubusercontent.com/5358146/100580958-e0daec80-330c-11eb-8362-4c6373f392bc.png">



<img width="1261" alt="Screenshot 2020-11-30 at 12 33 51 PM" src="https://user-images.githubusercontent.com/5358146/100580948-dcaecf00-330c-11eb-9270-c7289be2595b.png">


cc @apbendi / @mds1

Reply:
Make sense, using a chord see: https://github.com/gitcoinco/web/commit/e622584e31080c49110107c06df6e42d830914ae

Should resolve this issue